.changes/next-release/bugfix-Buffer-3b7f55e3.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "Buffer",
+  "description": "remove deprecated new Buffer() constructor in NodeJS"
```

`NodeJS` is officially called `Node.js`
```suggestion
  "description": "remove deprecated new Buffer() constructor in Node.js"
```